docs/source/HTF.rst

```diff
@@ -0,0 +1,54 @@
+
+..............................
+2020 July CAPE Case
+..............................
+
+For the 2020 CAPE case, we exercised the HTF to investigate the factors that lead to the low CAPE bias. This case was initialized on July 23, 2020 using GFS v15.2 and GFS v16. A large CAPE reservoir appeared over the Great Plains with a warm front passage over the northern Great Plains and no significant synoptic pattern impacting the Southern Great Plains (SGP). A comprehensive evaluation of GFS forecasts was made, focusing on the SGP and verifying against the observational dataset from the Atmospheric Radiation Measurement network. We investigated the large-scale advection fields over a 120 km × 120 km region centered at the SGP central facility site using European Centre for Medium-Range Weather Forecasts Reanalysis v5 (ERA5). Experiments were conducted using the Common Community Physics Package (CCPP) Single Column Model (SCM) to probe the impacts of initial conditions (IC) and surface forcing.
+
+Key findings for the CAPE case are listed below:
+
+- The lower CAPE in GFS is attributed to air at low levels being drier than observed.
+- The larger-than-observed Bowen ratio in GFS suggests an incorrect surface energy partitioning associated with soil moisture and soil temperature. 
+- GFS v16 exhibits more vertical mixing leading to a deeper boundary layer than v15.2 and observations.
+- The drier air in GFSv16 is mostly related to local physics not large-scale advection in this case.
+- With realistic IC of vertical profiles and surface fluxes, the GFS v16 physics suite is able to produce the large CAPE.
+
+..............................
+2020 Cold Air Damming
+..............................
+
+We adopted the latest version of Short-Range Weather (SRWeather) App (version 1.0) to investigate the 2020 CAD case1 developed as part of the UFS Case Studies project to investigate the impact of code updates, horizontal and vertical resolution, and physics suites. The SRWeather App is the foundation for building NOAA’s future convection-allowing Rapid Refresh Forecast System (RRFS). The limited area configuration needs fewer computer resources and can run models in high resolution and cycle frequently. Thus the case studies conducted with the SRWeather App are a useful supplement for the global configuration case studies. The physics suites used in this study, which focuses on the CONUS domain, include GFS v15.2, GFS v16, GFS v17α, and RRFS v1α. The results from UFS SRWeather App were compared with the results from the UFS global configuration. The SRWeather App was also tested with different configurations, such as different versions of the code base, model grid spacing (13 km vs 3 km, 64 levels vs 127 levels), and physics suite employed. The model forecasts were verified against station observations and analysis data. 
```

Comment:
```suggestion
We adopted the Short-Range Weather (SRWeather) App v 1.0 to investigate the 2020 CAD case to investigate the impact of code updates, horizontal and vertical resolution, and physics suites. The SRWeather App is the foundation for building NOAA’s future convection-allowing Rapid Refresh Forecast System (RRFS). The limited area configuration requires fewer computer resources so that the model can be run in high resolution and be cycled frequently. Thus the case studies conducted with the SRWeather App are a useful supplement for the global configuration case studies. The physics suites used in this study, which focuses on the CONUS domain, include GFS v15.2, GFS v16, GFS v17α, and RRFS v1α. The results from UFS SRWeather App were compared with the results from the UFS global configuration. The SRWeather App was also tested with different configurations, such as different versions of the code base, model grid spacing (13 km vs 3 km, 64 levels vs 127 levels), and physics suite employed. The model forecasts were verified against station observations and analysis data. 
```